build/npm/navigation-react-native/navigation.react.native.d.ts

```diff
@@ -525,6 +525,10 @@ export interface TabBarProps {
      * The default selected tab index
      */
     defaultTab?: number;
+    /**
+     * Indicates how labels should be displayed in the tab bar
+     */
+    labelVisibility?: "auto" | "labeled" | "unlabeled" | "selected";
```

Comment:
Do you think we should call it `labelVisibilityMode` instead of `labelVisibility` to match the android field? React Native sometimes use [mode to match android](https://reactnative.dev/docs/scrollview#overscrollmode-android)